packages/terra-compact-interactive-list/src/subcomponents/Cell.jsx

```diff
@@ -0,0 +1,95 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import ThemeContext from 'terra-theme-context';
+import { checkIfColumnIsResponsive } from '../utils/utils';
+import columnShape from '../proptypes/columnShape';
+import { widthUnitTypes } from '../utils/constants';
+import styles from './Cell.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * An identifier to uniquely identify the cell.
+   */
+  id: PropTypes.string.isRequired,
+
+  /**
+   * Content that will be rendered within the Cell.
+   */
+  children: PropTypes.node,
+
+  /**
+   * A column data.
+   */
+  column: columnShape,
+
+  /**
+   * Columns minimum width in units set by widthUnit prop, such as `px`, `em`, or `rem`.
+   */
+  columnMinimumWidth: PropTypes.number,
+
+  /**
+   * Columns maximum width in units set by widthUnit prop, such as `px`, `em`, or `rem`.
+   */
+  columnMaximumWidth: PropTypes.number,
+
+  /**
+   * The type of width value. One of `px`, `em`, `rem`. Defaults to 'px'.
+   */
+  widthUnit: PropTypes.oneOf([
```

Comment:
If you are going to allow column resizing, I think you must force consumers to use pixels.  Even if we can allow multiple units, I would not expect it to be separate from the width definition.  Usually there are not two separate props in CSS.

Reply:
Per what we discussed earlier with @eawww  there are no plans to implement the resizing in this component, and that `em` and `rem` are required.

As for `I would not expect it to be separate from the width definition. Usually there are not two separate props in CSS` - totally agree on that, however, due to unique needs of this component, its layout is heavily based on width calculations. My concern is that user might put "100px" for one column and "1.2em" for another column, which would break all calculations. Introducing that prop I was trying to ensure the way all width units are the same across the component, but open for better ways to do so.

Reply:
I removed widthUnit prop in https://github.com/cerner/terra-framework/pull/1869/commits/6f635dff9b8c8ee90cd59dfd9f5d3d61f8e42947. The customer will be allowed to pass all width props for columns (such as width, maxWidth, minWidth) with valid css string. I expect the customer pass all width props using the same width unit, the failed prop will be disregarded, which will not break the layout, just make the column flex growing. There will be also a console warning for that.